packages/core/src/Page/index.js

```diff
@@ -586,7 +586,7 @@ class Page {
 
     const renderedFooter = this.pageConfig.variableProcessor.renderSiteVariables(this.pageConfig.sourcePath,
                                                                                  footerContent, pageSources);
-    return `${pageData}\n${renderedFooter}`;
+    return `<div data-included-from="${this.footer}">${pageData}</div>\n${renderedFooter}`;
```

Comment:
👀

Reply:
let's add a test or two too: any relative link in one of the test_site layouts

Reply:
Ah, this was very careless of me. Thanks for spotting this!